wiki/Legal/Copyright/ru.md

```diff
@@ -1,61 +1,59 @@
 ---
 legal: true
-outdated: true
 ---
 
 # Политика osu! в отношении авторского права
 
-В соответствии с [Законом об авторском праве в цифровую эпоху](https://ru.wikipedia.org/wiki/Digital_Millennium_Copyright_Act), osu! сформировало следующую политику в отношении авторского права.
-Контактные данные представителя osu!, уполномоченного получать жалобы о нарушении авторских прав (далее — Представитель), указаны в конце данного документа и хранятся в архиве Бюро авторского права США (<https://www.copyright.gov/onlinesp/list/index.html>).
+osu! является игрой в значительной степени ориентированной на сообщество. Все игровые уровни (битмапы) создаются и загружаются участниками сообщества, чаще всего без получения разрешения от соответствующих правообладателей. Мы стараемся не рекламировать или получать прибыль непосредственно от загруженного контента, а игра полностью финансируется за счет пожертвований пользователей.
+
+Большая часть полученых пожертвований, которые мы получаем, используются для [лицензирования музыки](https://osu.ppy.sh/help/wiki/Legal/Music_Licensing). Запросы для лицензирования музыки могут быть отправлены на [music@ppy.sh](mailto:music@ppy.sh).
+
+В противном случае, osu! приняли следующую политику в отношении нарушения авторских прав в соответствии с Законом об авторском праве в цифровую эпоху, глава 17 статья 512 кодекса США.
+
+Контактные данные представителя osu!, уполномоченного получать жалобы о нарушении авторских прав (далее — Представитель), указаны в конце данного документа и хранятся в архиве Бюро авторского права США (<https://www.copyright.gov/onlinesp/list/index.html>). osu! серьёзно относится к защите авторских и других прав интеллектуальной собственности.
 
-osu! серьёзно относится к защите авторских и других прав интеллектуальной собственности.
 В политику osu! входят:
```

Comment:
this is different from line 33, but in English they say the same thing

Reply:
Has been left same as in the original version.  Thanks for reformatting